Modules/_testcapi/vectorcall.c

```diff
@@ -52,17 +60,22 @@ test_pyobject_fastcalldict(PyObject *self, PyObject *args)
     return PyObject_VectorcallDict(func, stack, nargs, kwargs);
 }
 
+/*[clinic input]
+_testcapi.pyobject_vectorcall
+    func: object
+    func_args: object
+    kwnames: object = None
```

Comment:
```suggestion
    kwnames: object
```

Sorry it was my fault :)